distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -273,6 +273,9 @@ get_constructor_name atom_constructor = @Builtin_Method "Meta.get_constructor_na
 new_atom : Any -> Array -> Atom
 new_atom constructor fields = @Builtin_Method "Meta.new_atom"
 
+lazy_atom : (Any -> Atom) -> Atom
+lazy_atom constructor fields = @Builtin_Method "Meta.lazy_atom"
```

Comment:
The name `lazy` isn't the best choice here – `lazy` suggests some sort of suspended value, while what we have here is just half-constructed. Really `atom_with_hole` is more descriptive imo

Reply:
Renamed to `atom_with_hole` in [9222ff5](https://github.com/enso-org/enso/pull/4023/commits/9222ff54c94035c7857eaba0fc8a3e5713cb0572)